types/googlemaps/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "npm-naming": false
+        "npm-naming": false,
+        "no-redundant-jsdoc-2": false
```

Comment:
What is this good for?

Reply:
We [already have deprecated tags](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/da25cdaf3a0581ba1db8300a469d29483720f4c2/types/googlemaps/reference/places-service.d.ts#L32-L33).

Reply:
This rules fails because of `@deprecated`.

```
The 'no-redundant-jsdoc-2' rule threw an error in '.../DefinitelyTyped/types/googlemaps/reference/map.d.ts':                                
Error: Unexpected tag kind: JSDocDeprecatedTag 
```

Reply:
I think this is the problem: microsoft/dtslint#299

Reply:
Please remove this change. You need to fix the error instead.

Reply:
>Please remove this change. You need to fix the error instead.

This appears to be an upstream issue that needs to be resolved. Please advise what I should do as the linting for this pr will fail without ignoring that rule.

Reply:
Why?

> **All checks have passed**. 
> 3 successful checks